main.js

```diff
@@ -0,0 +1,142 @@
+}
+
+btnOpenMenu.addEventListener('click', openMenu);
+btnCloseMenu.addEventListener('click', closeMenu);
+
+for (let i = 0; i < options.length; i += 1) {
+  options[i].addEventListener('click', closeMenu);
+}
+
+const data = {
+  project1: {
+    name: 'Tonic',
+    img: 'img/img1.png',
+    companyName: 'CANOPY',
+    job: 'Back End Dev',
+    year: 2015,
+    decription:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled is simply ',
+    tags: ['html', 'css', 'javaScript'],
+  },
+
+  project2: {
+    name: 'Multi-Post Stories',
+    img: 'img/img2.png',
+    companyName: 'FACEBOOK',
+    job: 'Full Stack Dev',
+    year: 2015,
+    decription:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ',
+    tags: ['html', 'Ruby on rails', 'css', 'javaScript'],
+  },
+
+  project3: {
+    name: 'Facebook 360',
+    img: 'img/img3.png',
+    companyName: 'FACEBOOK',
+    job: 'Full Stack Dev',
+    year: 2015,
+    decription:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ',
+    tags: ['html', 'Ruby on rails', 'css', 'javaScript'],
+  },
+  project4: {
+    name: 'Uber Navigation',
+    img: 'img/img4.png',
+    companyName: 'Uber',
+    job: 'Lead Developer',
+    year: 2018,
+    decription:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor  ',
+    tags: ['html', 'Ruby on rails', 'css', 'javaScript'],
+  },
+};
+
+const nameID = document.getElementById('name');
+const img = document.getElementById('img');
+const companyName = document.getElementById('companyName');
+const job = document.getElementById('job');
+const year = document.getElementById('data');
+const descrition = document.getElementById('description');
+const tags = document.getElementById('tags');
+const button1 = document.querySelector('#btn-1');
+const button2 = document.querySelector('#btn-2');
+const button3 = document.querySelector('#btn-3');
+const button4 = document.querySelector('#btn-4');
+const btnClosePop = document.querySelector('.popup-close-btn');
+const popupOverlay = document.querySelector('.popup-overlay');
+const popup = document.getElementById('popup');
+
+function openPopUp() {
+  popup.classList.remove('hidden');
+}
+
+function changePopup(order) {
+  tags.innerHTML = '';
+
+  const project = data[`project${order}`];
+  nameID.innerHTML = project.name;
+  img.src = project.img;
+  companyName.innerHTML = project.companyName;
+  job.innerHTML = project.job;
+  year.innerHTML = project.year;
+  descrition.innerHTML = project.decription;
+  const tagsLength = project.tags.length;
+  for (let i = 0; i < tagsLength; i += 1) {
+    const element = document.createElement('li');
+    element.innerHTML = `${project.tags[i]}`;
+    tags.appendChild(element);
+  }
+  openPopUp();
+}
```

Comment:
- [ ] Add responsive design for your popup, in the image below you can see how it looks like in a mobile version:
- [ ] Add more padding for your main content which contain all your information in your popup window, you can see that it is so close (check the yellow color in the image below)
- [ ] [OPTIONAL] Add the next CSS attribute to improve the UX in your app:
```
button:hover {
  cursor: pointer;
}
```

![image](https://user-images.githubusercontent.com/60164239/136291768-2788235b-8104-4e6f-a723-755714c45402.png)